subsys/bluetooth/host/Kconfig

```diff
@@ -55,7 +55,7 @@ config BT_HCI_TX_STACK_SIZE
 	default 256
 	default 256 if BT_H4
 	default 256 if BT_H5
-	default 256 if BT_SPI
+	default 416 if BT_SPI
```

Comment:
Well, I see that BT stacks are finely managed (e.g. 256 bytes here, 446 bytes for BT_CTLR_RX_PRIO_STACK_SIZE), that I followed that approach, just giving explanation why I didn't set it to "almost rount" 384 bytes. If it's really grounded, we can set it to 512. One concern is that BT_HCI_TX_STACK_SIZE isn't user-configurable, so not easy to optimize for a particular config (I actually plan to submit a ticket on that).